Jenkinsfile

```diff
@@ -7,7 +7,7 @@ pipeline {
 
   environment {
     REPO = 'audio_test_tools'
-    VIEW = getViewName(REPO)
```

Comment:
Shouldn't this change be undone?

Reply:
Another great spot - fixed in https://github.com/xmos/audio_test_tools/pull/192/commits/f292232adabbc348124b05a2d94d936b8ca0e0eb